### DISCUSSION TITLE
ability to scroll workspaces with a mouse

### DISCUSSION THREAD

**Discussion by froggy-jpg** [zen-browser/desktop#3415] [Ideas]
it would be great to be able to change your current workspace by hovering over workspaces with a mouse and using a scrollwheel

![image](https://github.com/user-attachments/assets/9c7d482e-016b-415a-add9-893651e838b1)

**Comment by Chaoscontrol** [comment]
Hmm I think you can do this. Just hold Ctrl and scroll anywhere over the sidebar. That's how you scroll through workspaces.
